manifests/install/os.pp

```diff
@@ -0,0 +1,13 @@
+# @summary Private class
+# @api private
+class apptainer::install::os {
+  assert_private()
+
+  if $facts['os']['family'] == 'RedHat' {
+    package{'apptainer':
```

Comment:
Will need to include `epel` module but could to gate behind `$manage_repo` parameter.

Reply:
Also would need to update metadata.json and .fixtures.yml to include EPEL module.